protocol-definitions/Jet.yaml

```diff
@@ -416,3 +416,28 @@ methods:
           nullable: false
           since: 2.6
           doc: 'True if processed successfully, false otherwise'
+  - id: 21
+    name: updateJobConfig
+    since: 2.6
+    doc: ''
+    request:
+      retryable: true
+      partitionIdentifier: -1
+      params:
+        - name: jobId
+          type: long
+          nullable: false
+          since: 2.6
+          doc: ''
+        - name: deltaConfig
+          type: Data
```

Comment:
just to double check here about using data here: we don't care about the backward compatibility for jet, right @viliam-durina?

Reply:
Right